src/pivotal-ui/components/meta/dropdown-driven-tabs.js

```diff
@@ -0,0 +1,14 @@
+// automatically attach a click listener to any dropdown-driven tab anchors on the page
+$(function () {
+  $('.dropdown-driven-tabs a').on('click', function (e) {
+    e.preventDefault()
+    var $a = $(this)
+    var $active = $a.closest('.dropdown-driven-tabs').find('.active')
+    if ($active) {
```

Comment:
Is this ever false? jQuery's `find` method will return an empty, truthy jquery object when it finds nothing.

You may want to make the whole thing be `$a.closest('.dropdown-driven-tabs').find('.active a').not($a).removeClass('active')`